cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -54,11 +54,17 @@ service PackagesService {
   //     put: "/core/packages/v1alpha1/installedpackages"
   //   };
   // }
-  // rpc DeleteInstalledPackage(DeleteInstalledPackageRequest) returns (DeleteInstalledPackageResponse) {
-  //   option (google.api.http) = {
-  //     delete: "/core/packages/v1alpha1/installedpackages"
-  //   };
-  // }
+  rpc DeleteInstalledPackage(DeleteInstalledPackageRequest) returns (DeleteInstalledPackageResponse) {
+    option (google.api.http) = {
+      // Example:
+      // /core/packages/v1alpha1/installedpackages/plugin/helm.packages/v1alpha1/ns/my-ns/release-identifier
```

Comment:
@antgamdia : I tried to use the same format that you'd used in the frontend URLs for the plugin: `helm.packages+v1alpha1` but the grpc transcoding basically enforces that one (path) segment maps to one variable. That's why I've done `{plugin.name}/{plugin.version}` . If you're OK with that, I might update the UI to use the same too so we're consistent.

Reply:
Yeah ok, I'm guessing you'll be touching that code soon anyway, thanks.